inbm/dispatcher-agent/dispatcher/aota/aota_command.py

```diff
@@ -50,6 +51,13 @@ def __init__(self,
                  dbs: ConfigDbs) -> None:
         # security assumption: parsed_manifest is already validated
 
+        if 'signature' in parsed_manifest and parsed_manifest['signature'] is not None:
+            self._signature = parsed_manifest['signature']
+        if 'hash_algorithm' in parsed_manifest and parsed_manifest['hash_algorithm'] is not None:
+            self._hash_algorithm = parsed_manifest['hash_algorithm']
+        else:
+            self._hash_algorithm = DEFAULT_HASH_ALGORITHM
```

Comment:
**Suggestion:** Refactor the conditional checks for 'signature' and 'hash_algorithm' to avoid repetition and improve readability. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._signature = parsed_manifest.get('signature')
        self._hash_algorithm = parsed_manifest.get('hash_algorithm', DEFAULT_HASH_ALGORITHM)
```